projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -262,7 +262,7 @@ public static function preset_exists( $preset ) {
 	 */
 	public static function get_preset( $preset ) {
 		return json_decode(
-			// phpcs:ignore WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents
+		// phpcs:ignore WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents
```

Comment:
Particularly this, why the bad indent?

Reply:
No idea, I guess the linter didn't like this and change it.